src/backends/coinmarketcap.ts

```diff
@@ -0,0 +1,88 @@
+import fetch from 'node-fetch'
+import BigNumber from 'bignumber.js'
+
+import { create as createLogger } from '../common/log'
+import { BackendInstance, BackendServices } from '../types/backend'
+import { AccountInfo } from '../types/accounts'
+const log = createLogger('coinmarketcap')
+
+export interface CMCBackendOpts {
+  spread: number
+  ratesApiUrl: string,
+}
+const RATES_API = 'https://api.coinmarketcap.com/v2/ticker/'
+
+export default class CMCBackend implements BackendInstance {
+  public rates: {}
+  protected spread: number
+  protected ratesApiUrl: string
+  protected getInfo: (accountId: any) => AccountInfo | undefined
+
+  constructor (opts: CMCBackendOpts, api: BackendServices) {
+    this.spread = opts.spread | 0
```

Comment:
`|` is a bitwise OR (which might actually work in this case since `undefined | 0 === 0`), but it should probably use a logical OR `||` instead for clarity.